test/specs/deployAndRetrieve.e2e.ts

```diff
@@ -459,6 +461,67 @@ describe('Deploy and Retrieve', async () => {
     expect(outputPanelText).toContain('ended SFDX: Deploy Source to Org');
   });
 
+  step('SFDX: Delete This from Project and Org', async () => {
+    const workbench = await (await browser.getWorkbench()).wait();
+    await utilities.getTextEditor(workbench, 'MyClass.cls');
+    // Run SFDX: Push Source to Default Org and Ignore Conflicts to be in sync with remote
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'SFDX: Push Source to Default Org and Ignore Conflicts',
+      10
+    );
+    // Clear the Output view first.
+    await utilities.runCommandFromCommandPrompt(workbench, 'View: Clear Output', 2);
+
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'SFDX: Delete This from Project and Org',
+      2
+    );
+
+    // Make sure we get a confirmation dialog
+    const confirmationDialogText =
+      'Deleting source files deletes the files from your computer and removes the corresponding metadata from your default org. Are you sure you want to delete this source from your project and your org?, source: Salesforce CLI Integration, notification, Inspect the response in the accessible view with Option+F2';
```

Comment:
I'm curious, how did you get the last part of `Inspect the response in the accessible view with Option+F2`?

This is what's in my inspector:
<img width="1728" alt="Screenshot 2024-04-08 at 2 16 08 PM (2)" src="https://github.com/forcedotcom/salesforcedx-vscode-automation-tests/assets/139700604/4391b3ea-1903-458c-9ab4-6bb4783e0da1">

Reply:
![image](https://github.com/forcedotcom/salesforcedx-vscode-automation-tests/assets/113132642/ed67f0e1-e2ce-481a-b645-f717f9c00bdb)